core/src/test/resources/org/bitcoinj/script/script_invalid.json

```diff
@@ -151,8 +151,8 @@
 ["2 2 LSHIFT", "8 EQUAL", "P2SH,STRICTENC", "disabled"],
 ["2 1 RSHIFT", "1 EQUAL", "P2SH,STRICTENC", "disabled"],
 
-["1","NOP1 NOP2 NOP3 NOP4 NOP5 NOP6 NOP7 NOP8 NOP9 NOP10 2 EQUAL", "P2SH,STRICTENC"],
```

Comment:
Regarding the merging of the script tests please see https://github.com/bitcoinj/bitcoinj/pull/1394